stdlib/public/runtime/CMakeLists.txt

```diff
@@ -74,15 +74,30 @@ add_swift_library(swiftRuntime IS_STDLIB IS_STDLIB_CORE
 foreach(sdk ${SWIFT_CONFIGURED_SDKS})
   if("${sdk}" STREQUAL "LINUX" OR "${sdk}" STREQUAL "FREEBSD")
     foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
-      set(arch_subdir "${SWIFT_SDK_${sdk}_LIB_SUBDIR}/${arch}")
+      if ("${sdk}" STREQUAL "LINUX" AND "${arch}" STREQUAL "armv7")
+        # Handle special case for ARM/Thumb on Linux.
+        # FIXME: Find a more elegant way to handle this case.
+        foreach(subarch armv7;armv7l;thumbv7;thumbv7l)
```

Comment:
There should be just one directory (these platforms are all the same), see my other comment.